joomla/joomla-cms
# Fix invalid markup

### Summary of Changes
The post-installation message is wrapped in a `<p>` element. When the message has block elements, it generates invalid HTML markup.

### Testing Instructions
Go to Components > Post-installation Messages
View page source
Find:
```

				<p class="small">
					Since version 3.5.0				</p>
				<p><p>Since Joomla! 3.5 a statistics plugin will submit anonymous data to the Joomla Project. This will only submit the Joomla version, PHP version, database engine and version, and server operating system.</p><p>This data is collected to ensure that future versions of Joomla can take advantage of the latest database and PHP features without affecting significant numbers of users. The need for this became clear when a minimum of PHP 5.3.10 was required when Joomla! 3.3 implemented the more secure Bcrypt passwords.</p><p>In the interest of full transparency and to help developers <a href="https://developer.joomla.org/about/stats.html">this data is publicly available.</a> An API and graphs will show the Joomla version, PHP versions and database engines in use.<br><br>If you do not wish to provide the Joomla Project with this information you can disable the plugin called System - Joomla Statistics.</p></p>
				<div>
															<a href="index.php?option=com_postinstall&amp;view=message&amp;task=unpublish&amp;id=3&amp;bb8fbd385e4c3cce23b936c5fa28591d=1" class="btn btn-inverse btn-small">
						Hide this message					</a>
									</div>
```

### Expected result
```
				<p class="small">
					Since version 3.5.0				</p>
				<div>
					<p>Since Joomla! 3.5 a statistics plugin will submit anonymous data to the Joomla Project. This will only submit the Joomla version, PHP version, database engine and version, and server operating system.</p><p>This data is collected to ensure that future versions of Joomla can take advantage of the latest database and PHP features without affecting significant numbers of users. The need for this became clear when a minimum of PHP 5.3.10 was required when Joomla! 3.3 implemented the more secure Bcrypt passwords.</p><p>In the interest of full transparency and to help developers <a href="https://developer.joomla.org/about/stats.html">this data is publicly available.</a> An API and graphs will show the Joomla version, PHP versions and database engines in use.</p><p>If you do not wish to provide the Joomla Project with this information you can disable the plugin called System - Joomla Statistics.</p>															<a href="index.php?option=com_postinstall&amp;view=message&amp;task=unpublish&amp;id=3&amp;bb8fbd385e4c3cce23b936c5fa28591d=1" class="btn btn-inverse btn-small">
						Hide this message					</a>
									</div>
```
